main.md

```diff
@@ -236,21 +235,17 @@ GET /authorize?response_type=code&client_id=s6BhdRkqt3
     &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb 
     &code_challenge_method=S256
     &code_challenge=K2-ltc83acc4h0c9w6ESC_rEMTJ3bww-uCHaoeK1t8U
-    &authorization_details=%5B%7B%22type%22%3A%22https%3A%2F%2F
-    www.someorg.com%2Fpayment_initiation%22%2C%22actions%22%3A%5
-    B%22initiate%22%2C%22status%22%2C%22cancel%22%5D%2C%22locat
-    ions%22%3A%5B%22https%3A%2F%2Fexample.com%2Fpayments%22%5D%
-    2C%22instructedAmount%22%3A%7B%22currency%22%3A%22EUR%22%2C
-    %22amount%22%3A%22123.50%22%7D%2C%22debtorAccount%22%3A%7B%
-    22iban%22%3A%22DE40100100103307118608%22%7D%2C%22creditorNa
-    me%22%3A%22Merchant123%22%2C%22creditorAccount%22%3A%7B%22i
-    ban%22%3A%22DE02100100109307118603%22%7D%2C%22remittanceInf
-    ormationUnstructured%22%3A%22Ref%20Number%20Merchant%22%7D%
-    5D%0A%20%20%20 HTTP/1.1
+    &authorization_details=%5B%7B%22type%22:%22account_information
```

Comment:
I'll admit that the exact rules for what characters need to be encoded with application/x-www-form-urlencoded are a bit mysterious to me. But I think that `/` and `:` are among them and those are unencoded here (and elsewhere). 

https://www.urldecoder.org/ gives the below and Java's `URLEncoder.encode(...)` agrees with that.  
`%5B%7B%22type%22%3A%22account_information%22%2C%22actions%22%3A%5B%22list_accounts%22%2C%22read_balances%22%2C%22read_transactions%22%5D%2C%22locations%22%3A%5B%22https%3A%2F%2Fexample.com%2Faccounts%22%5D%7D%5D`

Reply:
I couldn't find a definitiv answer whether / and : must be encoded but it's an option. I changed the examples to use this encoding.

Reply:
wfm